Yasgui/packages/yasqe/src/sparql.ts

```diff
@@ -74,7 +119,7 @@ export async function executeQuery(yasqe: Yasqe, config?: YasqeAjaxConfig): Prom
     return await req.then(
       (result) => {
         let hasMorePage = false;
-        if (yasqe.config.paginationOn) {
+        if ("update" !== yasqe.getQueryMode() && yasqe.config.paginationOn) {
```

Comment:
You can use the `isUpdateQuery` method from yasqe I believe.

Reply:
Hm. I was use it :) . Maybe I made mistake in some rebeising.